oneflow/core/vm/release_tensor_instruction_policy.h

```diff
@@ -130,60 +130,62 @@ class SlowReleaseTensorInstructionPolicy final : public ReleaseTensorInstruction
 
 struct MakeReleaseTensorInstructionPolicy
     : public StreamRoleVisitor<MakeReleaseTensorInstructionPolicy> {
-  static std::unique_ptr<vm::InstructionPolicy> VisitCompute(
+  static Maybe<vm::InstructionPolicy> VisitCompute(
       DataType data_type, const std::shared_ptr<vm::EagerBlobObject>& eager_blob_object,
       const Optional<vm::Stream*>& stream) {
     return Make(data_type, eager_blob_object, stream);
   }
-  static std::unique_ptr<vm::InstructionPolicy> VisitHost2Device(
+  static Maybe<vm::InstructionPolicy> VisitHost2Device(
       DataType data_type, const std::shared_ptr<vm::EagerBlobObject>& eager_blob_object,
       const Optional<vm::Stream*>& stream) {
     return Make(data_type, eager_blob_object, stream);
   }
-  static std::unique_ptr<vm::InstructionPolicy> VisitDevice2Host(
+  static Maybe<vm::InstructionPolicy> VisitDevice2Host(
       DataType data_type, const std::shared_ptr<vm::EagerBlobObject>& eager_blob_object,
       const Optional<vm::Stream*>& stream) {
     return Make(data_type, eager_blob_object, stream);
   }
-  static std::unique_ptr<vm::InstructionPolicy> VisitSyncedLaunchedCommNet(
+  static Maybe<vm::InstructionPolicy> VisitSyncedLaunchedCommNet(
       DataType data_type, const std::shared_ptr<vm::EagerBlobObject>& eager_blob_object,
       const Optional<vm::Stream*>& stream) {
     return Make(data_type, eager_blob_object, stream);
   }
-  static std::unique_ptr<vm::InstructionPolicy> VisitAsyncedLaunchedCommNet(
+  static Maybe<vm::InstructionPolicy> VisitAsyncedLaunchedCommNet(
       DataType data_type, const std::shared_ptr<vm::EagerBlobObject>& eager_blob_object,
       const Optional<vm::Stream*>& stream) {
     return Make(data_type, eager_blob_object, stream);
   }
-  static std::unique_ptr<vm::InstructionPolicy> VisitBarrier(
+  static Maybe<vm::InstructionPolicy> VisitBarrier(
       DataType data_type, const std::shared_ptr<vm::EagerBlobObject>& eager_blob_object,
       const Optional<vm::Stream*>& stream) {
     UNIMPLEMENTED();
```

Comment:
UNIMPLEMENTED_THEN_RETURN() << "ReleaseTensor instruction not supported in Barrier stream";